src/service/world-api-adapter.js

```diff
@@ -1,11 +1,22 @@
 /**
  * ##World API Adapter
  *
- * The World API Adapter allows you to create, access, and manipulate multiplayer worlds its users and runs
+ * Epicenter includes support for multiplayer games. These are projects where multiple end users share a run and work together in the same "world." Only [team projects](../../../glossary/#team) can be multiplayer.
  *
- * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
+ * The World API Adapter allows you to create, access, and manipulate multiplayer worlds, including end users and runs playing within the world.
  *
+ * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
  *
+ * Typically, you instantiate a World Adapter and then access the methods provided. Instantiating requires the account (**Team ID**), project (**Project ID**), and group (**Group Name**).
```

Comment:
all apis use "account id", but in the UI it's either the "team id" or the "user id" (that is, the thing that's called user id when you set it up in the UI, not the guid for the user). so usually i put "team id" in parens and/or in notes, depending on what format we're using. for example in https://forio.com/epicenter/docs/public/aggregate_run_api/ we say: "The Account ID, as part of the URI. This is the "Team ID" (for team projects) or "User ID" (for personal projects)."